android/src/main/java/io/mosip/tuvali/ble/central/impl/GattClient.kt

```diff
@@ -22,7 +22,7 @@ class GattClient(var context: Context) {
   private lateinit var onDeviceConnected: (BluetoothDevice) -> Unit
   private var peripheral: BluetoothDevice? = null
   private var bluetoothGatt: BluetoothGatt? = null
-  private val logTag = getLogTag("BLECentral")
+  private val logTag = getLogTag((this::class.java.simpleName).toString())
```

Comment:
This can be simplified to `this.className.simpleName` in all places

Reply:
Krishna, This  code  this.className.simpleName is not valid in Kotlin, we can use -- this::class.simpleName -- or since we are accessing inside the class itself we can also use  -- ::class.simpleName --. But readability is good in this --this::class.simpleName -- one compared to other one.

Reply:
We do not need toString on this::class.java.simpleName as simpleName returns string. So we can avoid converting to string again. Also instead of `this::class.java.simpleName` you can use `javaClass.simpleName` to make it simplified.

Reply:
Also, please make changes in all the files. The changes are not looking consistent. I can see string in logTags still now in some of the classes.

Reply:
done